cves/kernel/CVE-2019-12378.yml

```diff
@@ -358,8 +357,8 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: Forgot null pointer check, that is the fix to the bug.
```

Comment:
Just a stylistic grammar thing, I'd say "which was" instead of "that is."

Reply:
updated